litmus-portal/frontend/src/pages/MonitoringDashboardPage/styles.ts

```diff
@@ -111,6 +121,211 @@ const useStyles = makeStyles((theme) => ({
   modal: {
     padding: theme.spacing(15, 0),
   },
+
+  selectDate: {
+    display: 'flex',
+    height: '2.9rem',
+    minWidth: '9rem',
+    marginLeft: theme.spacing(3.75),
+    border: `0.1px solid ${theme.palette.border.main}`,
+    background: theme.palette.background.paper,
+    borderRadius: 4,
+    textTransform: 'none',
+  },
+
+  displayDate: {
+    width: '100%',
+    color: theme.palette.text.primary,
+  },
+
+  rangeSelectorIcon: {
+    width: '0.625rem',
+    height: '0.625rem',
+    marginLeft: theme.spacing(1),
+    marginRight: theme.spacing(-1),
+  },
+
+  rangeSelectorClockIcon: {
+    width: '0.825rem',
+    height: '0.825rem',
+    marginLeft: theme.spacing(-1),
+    marginRight: theme.spacing(1),
+  },
+
+  rangeSelectorPopover: {
+    marginTop: theme.spacing(37.5),
+  },
+
+  formControl: {
+    width: '9rem',
+    marginLeft: theme.spacing(1.5),
+    [theme.breakpoints.down('sm')]: {
+      marginLeft: theme.spacing(3.75),
+    },
+    '& .MuiSelect-outlined': {
+      padding: '0.925rem',
+      '&:focus': {
+        borderColor: theme.palette.primary.main,
+      },
+      '& .MuiInputLabel-root': {
+        color: `${theme.palette.text.hint} !important`,
+        marginTop: `${theme.spacing(2)} !important`,
+      },
+    },
+  },
+
+  inputLabel: {
+    color: theme.palette.text.hint,
+    marginTop: theme.spacing(-1),
+    '&.MuiInputLabel-shrink': {
+      marginTop: theme.spacing(1),
+    },
+  },
+
+  menuListItem: {
+    color: theme.palette.text.hint,
+    height: '2.4rem',
+  },
+
+  menuListItemSelected: {
+    height: '2.4rem',
+    background: `${theme.palette.primary.light} !important`,
+    color: theme.palette.secondary.contrastText,
+    '&:hover': {
+      background: `${theme.palette.primary.light} !important`,
+    },
+  },
+
+  refreshIcon: {
+    marginRight: theme.spacing(1),
+  },
+
+  refreshText: {
+    marginTop: theme.spacing(-2.85),
+    marginLeft: theme.spacing(3.5),
+  },
+
+  headerInfoText: {
+    fontWeight: 500,
+    fontSize: '0.95rem',
+    lineHeight: '1rem',
+    width: '35%',
+    marginLeft: theme.spacing(2.5),
+  },
+
+  controls: {
+    marginRight: theme.spacing(2.25),
+    display: 'flex',
+    [theme.breakpoints.down('sm')]: {
+      flexDirection: 'column',
+      marginRight: 0,
+    },
+  },
+
+  chaosTableSection: {
+    padding: theme.spacing(2.5, 2, 0),
+  },
+
+  accordionSummary: {
+    display: 'flex',
+    justifyItems: 'center',
+    background: theme.palette.disabledBackground,
+  },
+
+  accordionDetails: {
+    width: '100%',
+  },
+
+  chaosHelperText: {
+    fontWeight: 500,
+    fontSize: '1rem',
+    color: theme.palette.primary.main,
+  },
+
+  tableDropIcon: {
+    width: '1.75rem',
+    height: '1.75rem',
+    marginTop: theme.spacing(-0.15),
+    color: theme.palette.primary.main,
+  },
+
+  accordionHeader: {
+    display: 'flex',
+  },
+
+  editIconButton: {
+    marginTop: theme.spacing(-0.75),
```

Comment:
same

Reply:
we do use negative margins, its required here.

Reply:
same as above.